client/components/date-picker/index.jsx

```diff
@@ -194,7 +194,7 @@ class DatePicker extends PureComponent {
 				renderDay={ this.renderDay }
 				localeUtils={ this.locale() }
 				onMonthChange={ this.props.onMonthChange }
-				enableOutsideDays={ this.props.enableOutsideDays }
+				showOutsideDays={ this.props.enableOutsideDays }
```

Comment:
we probably want to update the passed prop too to match the new name.

Reply:
agreed